apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/RDATestUtils.java

```diff
@@ -284,7 +284,7 @@ private RdaFissClaim fissTestDataB(Mbi mbi) {
             .dcn(FISS_CLAIM_B_DCN)
             .intermediaryNb("99999")
             .hicNo("hicnumbe2")
-            .currStatus('\0')
+            .currStatus('0')
```

Comment:
This caused an issue when running the test with postgres, since it read it as a null value and postgres doesnt allow it for the field type. I talked about it with Brian and we suspect it was a typo anyhow.

Reply:
Good example of the difference between hsql and postgres and how it could affect a test case (and therefore our ability to catch a bug) though.